example/FormatResp.py

```diff
@@ -6,57 +6,68 @@
 # This source code is licensed under Apache 2.0 License.
 
 
+from typing import Dict
+
+import pandas as pd
 import prettytable
+from nebula3.data.DataObject import Value, ValueWrapper
+from nebula3.data.ResultSet import ResultSet
+
+
+################################
+#     Method 1 (Recommended)   #
+################################
+def result_to_df(result: ResultSet) -> pd.DataFrame:
+    """
+    build list for each column, and transform to dataframe
+    """
+    assert result.is_succeeded()
+    columns = result.keys()
+    d: Dict[str, list] = {}
+    for col_num in range(result.col_size()):
+        col_name = columns[col_num]
+        col_list = result.column_values(col_name)
+        d[col_name] = [x.cast() for x in col_list]
+    return pd.DataFrame.from_dict(d, columns=columns)
+
+
+################################
+#     Method 2   (Customize)   #
+################################
+cast_as = {
+    Value.NVAL: "as_null",
+    Value.__EMPTY__: "as_empty",
+    Value.BVAL: "as_bool",
+    Value.IVAL: "as_int",
+    Value.FVAL: "as_double",
+    Value.SVAL: "as_string",
+    Value.LVAL: "as_list",
+    Value.UVAL: "as_set",
+    Value.MVAL: "as_map",
+    Value.TVAL: "as_time",
+    Value.DVAL: "as_date",
+    Value.DTVAL: "as_datetime",
+    Value.VVAL: "as_vertex",
+    Value.EVAL: "as_edge",
+    Value.PVAL: "as_path",
+    Value.GGVAL: "as_geography",
+    Value.DUVAL: "as_duration",
+}
+
+
+def customized_cast_with_dict(val: ValueWrapper):
+    _type = val._value.getType()
+    return getattr(_type, cast_as(_type))()
```

Comment:
this line may cause an error
if no attr will return None
and None() will cause the error.
do we need to consider that?

Reply:
@yihong0618  I try to handle error in the [latest commit](https://github.com/vesoft-inc/nebula-python/pull/222/commits/471a633d96b0ec3be66e3806b102916afbd3c0fc).